ttnn/cpp/ttnn/operations/transformer/sdpa_decode_gqa/sdpa_decode_gqa.hpp

```diff
@@ -18,6 +18,9 @@ struct ExecuteScaledDotProductAttentionGQADecode {
         const ttnn::Tensor &input_tensor_k,
         const ttnn::Tensor &input_tensor_v,
         const std::vector<uint32_t> cur_pos,
+        const std::optional<const Tensor> cur_pos_tensor = std::nullopt,
+        std::optional<bool> transpose_q = true,
+        std::optional<bool> share_cache = false,
```

Comment:
Should these just be regular args with default values?

Reply:
Good point! My next pr actually changed it to regular args with default values.